internal/pkg/levee/levee.go

```diff
@@ -94,20 +118,77 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
+func doPointerAnalysis(pass *analysis.Pass, analysisConf *config.Config, c *ssa.Call) (isSafeCall bool) {
+	pkg := c.Parent().Pkg
+	main := pkg.Func("main")
+	if main == nil {
+		return false
+	}
+
+	pointerConf := &pointer.Config{
+		Mains: []*ssa.Package{pkg},
+	}
+
+	vs := callValues(c)
+	if len(vs) == 0 {
+		return false
+	}
+
+	for _, v := range vs {
+		if !pointer.CanPoint(v.Type()) {
+			continue
+		}
+		pointerConf.AddQuery(v)
+	}
+
+	result, err := pointer.Analyze(pointerConf)
+	if err != nil {
+		return false
+	}
+
+	isSafeCall = true
+	for _, pSet := range result.Queries {
```

Comment:
Since documentation here https://godoc.org/golang.org/x/tools/go/pointer is very terse, do you mind putting a few comments that explain the flow?

Reply:
Sure. Done, let me know if it's better now.